scripts/test/Muon/help_widget_presenter_test.py

```diff
@@ -25,11 +25,11 @@ def setUp(self):
     def tearDown(self):
         self.view = None
 
-    @mock.patch('Muon.GUI.Common.help_widget.help_widget_view.ManageUserDirectories')
+    @mock.patch('Muon.GUI.Common.help_widget.help_widget_view.manageuserdirectories')
     def test_that_manage_directories_button_clicked_opens_directory_manager(self, ManageUserDirectories_mock):
         self.view.manage_user_dir_button.clicked.emit(True)
 
-        ManageUserDirectories_mock.openUserDirsDialog.assert_called_once_with(self.view)
+        ManageUserDirectories_mock.open_mud.assert_called_once_with(self.view)
```

Comment:
Could we add a test which simulates opening it multiple times to guarantee this behaviour does not regress